ocaml/xapi/repository_helpers.ml

```diff
@@ -1151,8 +1144,15 @@ let eval_guidances ~updates_info ~updates ~kind =
   |> GuidanceSet.resort_guidances ~kind
   |> GuidanceSet.elements
 
+let repoquery_sep = ":|"
+
+let get_repoquery_fmt () =
```

Comment:
This doesn't actually need to be a function, as it is entirely static (so `()` can be omitted). But it doesn't matter much.